lambda_function.py

```diff
@@ -16,9 +16,9 @@ def lambda_handler(event, context):
     """
     print(event)
```

Comment:
Do we really need to print the event in this use case? If so we should use the python logging library instead of print statements . it makes it easier to detect the line that's calling the log statement, has severity ratings(I.e warning, debug info). Something like this will work:
```
import logging
logging.info(event)
```

Reply:
It's been a while but this was likely just a troubleshooting mechanism that got left in. It can probably be removed

Reply:
Noted. I didn't add this but I'll use the logging method for proper, non-troubleshooting purposes. This will probably get deleted but we used it as recently as today for troubleshooting.